sdk/src/signer.rs

```diff
@@ -34,6 +34,18 @@ pub trait Signer {
         None
     }
 
+    /// Request RFC 3161 timestamp to be included in the manifest data
+    /// structure.
+    ///
+    /// `message` is a preliminary hash of the claim
+    ///
+    /// The default implementation will send the request to the URL
+    /// provided by [`Self::time_authority_url()`], if any.
+    fn send_rfc3161_request(&self, message: &[u8]) -> Option<Result<Vec<u8>>> {
```

Comment:
I'm not sure I like the idea of the RFC reference being in the function name here. I'd agree it is technically correct and specific, but I think simply describing what the function does, rather than requiring them to look up an RFC is much clearer overall. It would be different if there were multiple protocols we were supporting and we needed to make clear which this one is.

Reply:
@gpeacock good suggestion. I've changed the name to `send_timestamp_request`.